src/fileActions.jsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from 'cockpit';
+import { useDialogs } from "dialogs.jsx";
+import React, { useState } from 'react';
+import {
+    Button,
+    Form, FormGroup,
+    Modal,
+    Stack,
+    TextInput,
+} from "@patternfly/react-core";
+
+import { InlineNotification } from "../pkg/lib/cockpit-components-inline-notification";
```

Comment:
Please be consistent on the pattern of importing. Above a few lines you import useDialogs from pkg/lib/dialogs. pkg/lib is one of the default import paths, so no needed to explicitely specify, so maybe be consistent about it, and also keep the order of imports a bit consistent.

First cockpit;
Then imports node_modules
Then local JSX imports
Tehn CSS imports